projects/bulma-app/src/app/tab/tab.component.html

```diff
@@ -0,0 +1,6 @@
+<bu-tabs>
+  <bu-tab-item title="Home" >
```

Comment:
can you add some content within the tab to check if it renders it correctly?

Reply:
that's the reason I asked this question on original issue : Are you expecting tab content as a part of this implementation, because i am not able to see any example on bulma site which shows tab as well as tab content.

Reply:
Adding just content will not work, we need to implement lot of things around it, show /hide according to tab, need to add directive which will accept the tab content as a template

something like this, what you say?

`<bu-tabs>
  <bu-tab-item title="Music">
    <ng-template bu-tab-content>
      dummy tab content
    </ng-template>
  </bu-tab-item>
</bu-tabs>`

Reply:
Yes we need fully functional tab. Yes the documentation does not contain the correct example as of now.